tests/dragonfly/acl_family_test.py

```diff
@@ -189,3 +189,36 @@ async def test_acl_categories_multi_exec_squash(df_local_factory):
 
     await admin_client.close()
     await client.close()
+
+
+@pytest.mark.asyncio
+async def test_acl_deluser(df_local_factory):
```

Comment:
These pass, but the above issues remain. What happens if we are executing a transaction and we suddenly "nuke" the connection. What happens to the context and the transaction?